stdlib/3/dbm/gnu.pyi

```diff
@@ -0,0 +1,31 @@
+
+from typing import Union, MutableMapping, Optional, Type, Iterator
+from types import TracebackType
+
+_KeyType = Union[str, bytes]
+_ValueType = Union[str, bytes]
+
+class error(OSError): ...
+
+# Actual typename gdbm, not exposed by the implementation
+class _gdbm(MutableMapping[_KeyType, _ValueType]):
```

Comment:
I'm not sure the MutableMapping base class makes sense here, because it doesn't fully satisfy the interface and doesn't inherit from MutableMapping at runtime.

Also, missing the `keys` method which returns `List[bytes]`.